### PR TITLE
Chore/setup base style

### PR DESCRIPTION
**Pull request by junan**
## What happened

Fixed [4](https://github.com/junan/nimble-survey/issues/4)
 
## Insight

Added
-  variables
-  typography
-  colors
-  assets
 
## Proof Of Work

Boostrap is working and icons are loading 

Screen Shot 2020-11-19 at 6 33 30 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,12 +19,16 @@
     "@angular/compiler": "11.0.0",
     "@angular/core": "11.0.0",
     "@angular/forms": "11.0.0",
+    "@angular/localize": "11.0.0",
     "@angular/platform-browser": "11.0.0",
     "@angular/platform-browser-dynamic": "11.0.0",
     "@angular/router": "11.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^8.0.0",
```
Comment (Lahphim) [inline package.json:26]: @junan I think it better to install specific version 😅

File: package.json [hunk 2]
```diff
@@ -18,12 +19,16 @@
...
+    "bootstrap": "4.5.0",
     "express": "4.17.1",
     "path": "0.12.7",
     "rxjs": "6.6.0",
+    "stylelint-scss": "^3.18.0",
```
Comment (Lahphim) [inline package.json:31]: @junan Here too 🙌

File: src/stylesheets/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  100: hsl(210, 50%, 98%), // #f9fbfd
+  200: hsl(206, 26%, 95%), // #eef2f5
+  300: hsl(214, 16%, 92%), // #e6e9ed
+  400: hsl(224, 10%, 79%), // #c3c6ce
+  500: hsl(227, 8%, 65%), // #9ea1ac
+  600: hsl(229, 6%, 49%), // #747784
+  700: hsl(227, 8%, 33%), // #4d505b
+  800: hsl(230, 9%, 25%), // #3b3d47
+  900: hsl(234, 11%, 18%), // #292a33
+  backgound: hsl(240, 11%, 9%) //#15151a
```
Comment (Lahphim) [inline src/stylesheets/_variables.scss:12]: @junan What if we naming with this http://chir.ag/projects/name-that-color/

File: src/stylesheets/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+$white: #fff;
+$black: #000;
+$gray: map-get($brand-gray, 500);
+
+$primary: $white;
+$secondary: $gray;
+
+// Fonts
+$font-size-base: map-get($font-size, "md");
```
Comment (Lahphim) [inline src/stylesheets/_variables.scss:43]: @junan Should it be just `'` instead of `"`? 🤔

File: src/styles.scss [hunk 5]
```diff
@@ -1 +1 @@
-/* You can add global styles to this file, and also import other style files */
+@import "./stylesheets/application";
```
Comment (Lahphim) [inline src/styles.scss:1]: @junan I saw here https://github.com/junan/nimble-survey/pull/22/files#diff-30f135108dde64f0b8b9f0df3f0e69a4a547b48557a13bd682797cfc54128f4fR4 using `'`, then I think we need to make it the same? 🤔

**Comment by Lahphim** [comment]
@junan What if we create other task separate about managing the icon as icon-sprite 🤔

**Comment by junan** [comment]
> @junan What if we create other task separate about managing the icon as icon-sprite? 🤔

@Lahphim Done

**Review by Lahphim — APPROVED** [review]

File: package.json [hunk 6]
```diff
@@ -18,12 +19,16 @@
...
+    "@angular/localize": "11.0.0",
     "@angular/platform-browser": "11.0.0",
     "@angular/platform-browser-dynamic": "11.0.0",
     "@angular/router": "11.0.0",
+    "@ng-bootstrap/ng-bootstrap": "8.0.0",
+    "bootstrap": "4.5.0",
     "express": "4.17.1",
     "path": "0.12.7",
     "rxjs": "6.6.0",
+    "stylelint-scss": "3.18.0",
```
Comment (carryall) [inline package.json:31]: @junan How about moving this to `devDependencies`?

File: src/stylesheets/_variables.scss [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+$white: #fff;
+$black: #000;
+$red: map-get($brand-red, 'primary');
+$blue: map-get($brand-blue, 'primary');
+$gray: map-get($brand-gray, 500);
+
+$primary: $white;
+$secondary: $gray;
+
+// Fonts
```
Comment (carryall) [inline src/stylesheets/_variables.scss:53]: @junan How about adding `$font-family-base` here too so we can set the base font family?

**Review by carryall — APPROVED** [review]

**Pull request merged**